src/code/z_actor.c

```diff
@@ -1542,25 +1542,37 @@ PosRot Actor_GetWorldPosShapeRot(Actor* actor) {
     return worldPosRot;
 }
 
-f32 func_8002EFC0(Actor* actor, Player* player, s16 arg2) {
-    s16 yawTemp = (s16)(actor->yawTowardsPlayer - 0x8000) - arg2;
+/**
+ * Returns the squared xyz distance from the actor to Player.
+ *
+ * This distance will be adjusted smaller if Player is already targeting an actor.
+ * In this case, the more Player is facing the actor, the smaller the distance is adjusted.
+ */
+f32 Target_AdjustedDistToPlayerSq(Actor* actor, Player* player, s16 playerShapeYaw) {
+    s16 yawTemp = (s16)(actor->yawTowardsPlayer - 0x8000) - playerShapeYaw;
     s16 yawTempAbs = ABS(yawTemp);
 
     if (player->unk_664 != NULL) {
         if ((yawTempAbs > 0x4000) || (actor->flags & ACTOR_FLAG_27)) {
             return MAXFLOAT;
         } else {
-            f32 ret =
+            f32 adjDistSq;
+
+            // Linear scaling, yaw being 90 degrees means it will return the original distance.
+            // 0 degrees will adjust to 60% of the distance.
+            adjDistSq =
                 actor->xyzDistToPlayerSq - actor->xyzDistToPlayerSq * 0.8f * ((0x4000 - yawTempAbs) * (1.0f / 0x8000));
 
-            return ret;
+            return adjDistSq;
         }
     }
 
+    // An actor will not be considered targetable if Player is facing more than ~60 degrees away
     if (yawTempAbs > 0x2AAA) {
```

Comment:
does `DEG_TO_BINANG(60)` work?

Reply:
Currently the codebase opts not to show any angle constants in degrees. This was a huge debate back in the day with plenty of arguments on both sides. I would love to show them in degrees as it feels more user friendly, but some of the math oriented folks here hate degrees and would rather show as radians, for example. 

So basically even if it did work, I wouldnt want to change it for just this one until we come to a decision on that topic as a whole

Reply:
oh, we've started doing it anyway (kinda) in #2047

Fair enough on punting on this though.,I'll try to search Discord for the context. Though using radians seems user-hostile to me, the first thing I would do seeing "pi/6 radians" is convert it to degrees in my head.